src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx

```diff
@@ -0,0 +1,124 @@
+
+With New Relic's account structure you can:
+
+* Maximize data ingest
+* Keep data organized 
+* Monitor multiple environments, like staging and development
+* Manage billing across different areas of your organization
+
+This doc will guide you through setting up your account structure. If you're going to create multiple accounts, it is best to do that _before_ you begin ingesting data. It can be harder to make changes after you connect New Relic to your applications because you risk losing historical context.
+
+<Callout variant="tip">
+If you're using Free or Standard editions, you only have one account available and you won't need this doc. Feel free to revisit this page if you decide to change to [Pro or Enterprise](https://newrelic.com/pricing)
+</Callout>
+
+## Before you get started [#begin]
+
+In this guide we're going to help you create and manage your New Relic account structure. 
+
+First, make sure you're familiar with our terms:
+
+* **Organization**:  An organization represents a New Relic customer. Your New Relic organization contains your organization's accounts, users, and data.
+* **Account**: When you sign up for New Relic your organization is automatically assigned one account. An account can have many sources of data reporting to it. For example, a single account could have data reporting from our infrastructure agent, an APM agent, and other integrations. All data reported to New Relic requires an account ID, which tells New Relic which account that data belongs in. That ID is also used for some account-specific tasks, like making API calls. 
+* **User**: In your organization, your [New Relic users](/docs/accounts/accounts-billing/new-relic-one-user-management/introduction-managing-users/) are granted access to specific accounts that are relevant to their duties and responsibilities. To manage users' access to accounts, you grant groups access to specific roles on specific accounts. Our user management system allows you to create the user access you need, whether that's a relatively simple setup with just a few roles across a few accounts, or a complex one with many roles across many accounts. 
+
+## Single account structure [#single-account-structure]
+
+Signing up for New Relic automatically generates an organization. An organization contains one account unless you manually create more. One account can be perfect for many companies and it might not be necessary for you to add more. 
+
+Some indications that one account is right for you:
+
+* You have 200 employees or less and you don't anticipate growing too much
```

Comment:
It seems like there could be many exceptions to the rule-of-thumb. Are there other factors that combine with the 200 number that makes this clearer?